semgrep/tests/e2e/snapshots/test_cli_test/test_timeout/results.json

```diff
@@ -0,0 +1,7 @@
+{
+  "config_missing_tests": [
+    "rules/cli_test/error/long.yaml"
```

Comment:
Why a timeout on a file should generate a config_missing_tests in the JSON??
In fact the rules/cli_test/error/long.py file is useless, because it's in the rules/ directory, and run_semgrep_in_tmp
will take target relative to e2e/targets/ ...

This test never worked as intended.

Reply:
I've fixed it in https://github.com/semgrep/semgrep/pull/10189